packages/utils/src/surescripts/bundle-verification.ts

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env node
```

Comment:
How are you using this? AFAIK, we either run the scripts w/ `npm run script`, `ts-node <path-to-script`, or rename "node" above to "ts-node" and add execution access to this file in order to run it directly.

TBH, I don't think it's worth it, as calling w/ `ts-node` is super straightforward, but just trying to make sure we're adding only what's needed.